docs/section19/19-2.md

```diff
@@ -0,0 +1,74 @@
+- Cycle through each terminal unit in the B-RMR and create list of HW loops: `for terminal in B_RMR...Terminal:`  
+    - Reset list with the one terminal unit: `terminal_list = []` 
+    - Check if heating source is hot water: `if terminal.heating_source == "HOT_WATER":`  
+        - Create a list with the one terminal unit: `terminal_list = [terminal.id]`  
+        - Check if the loop is attached to a boiler: `if are_all_terminal_heating_loops_attached_to_boiler(B_RMR,terminal_list) ==  TRUE:`  
+            - Check if the fluid loop is already in the list of HW loops, if not then add: `if terminal.heating_from_loop Not in HW_fluid_loop_list: HW_fluid_loop_list.append(terminal.heating_from_loop)`  
+
+- Eliminate duplicates from all loop lists: `HW_fluid_loop_list = List(Set(HW_fluid_loop_list))`  
+- Eliminate duplicates from all loop lists: `CHW_fluid_loop_list = List(Set(CHW_fluid_loop_list))`  
+- Eliminate duplicates from all loop lists: `CW_fluid_loop_list = List(Set(CW_fluid_loop_list))`  
+
+**Applicability Check 1:**   
+- For each fluid_loop in B_RMR.ASHRAE229: `For fluid_loop in B_RMR.ASHRAE229:`  
+    - Check if the loop is in any of the lists created above, if it is carry on and if not then this rule is Not_Applicable for the fluid loop: `if fluid_loop in HW_fluid_loop_list OR fluid_loop in CHW_fluid_loop_list OR fluid_loop in CW_fluid_loop_list:`  
+        - Reset FluidLoopDesignAndControl_cool_b: `FluidLoopDesignAndControl_cool_b = ""`
+        - Reset FluidLoopDesignAndControl_heat_b: `FluidLoopDesignAndControl_heat_b = ""`  
+        - if the loop is in the CHW fluid loop: `if fluid_loop in CHW_fluid_loop_list:`  
+            - Get cooling_or_condensing_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_cool_b = fluid_loop.cooling_or_condensing_design_and_control`  
+        - Elif the loop is in the CW fluid loop: `if fluid_loop in CW_fluid_loop_list:`  
+            - Get cooling_or_condensing_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_cool_b = fluid_loop.cooling_or_condensing_design_and_control`  
+        - Elif the loop is in the HW fluid loop: `if fluid_loop in HW_fluid_loop_list:`  
+            - Get heating_design_and_control FluidLoopDesignAndControl object: `FluidLoopDesignAndControl_heat_b = fluid_loop.heating_design_and_control`  
+
+        - **Rule Assertion:** 
+        - Case 1: If FluidLoopDesignAndControl_cool_b.is_sized_using_coincident_load = TRUE and FluidLoopDesignAndControl_heat_b = "" then pass: `if FluidLoopDesignAndControl_cool_b.is_sized_using_coincident_load == TRUE AND FluidLoopDesignAndControl_heat_b == "": outcome = "PASS"`  
+        - Case 2: Else if FluidLoopDesignAndControl_heat_b.is_sized_using_coincident_load = TRUE and FluidLoopDesignAndControl_cool_b = "" then pass: `if FluidLoopDesignAndControl_heat_b.is_sized_using_coincident_load == TRUE AND FluidLoopDesignAndControl_cool_b == "": outcome = "PASS"`  
```

Comment:
Could we have just one parameter: `fluid_loop_design_and_control` and use `type` under the `fluid_loop` to distinguish heat and cool?

This could be possibly rewrite as:

```python
is_sized_using_coincident_load = False
if fluid_loop.type == COOLING and fluid_loop_id in CHW_fluid_loop_list:
    is_sized_using_coincident_load = fluid_loop.cooling_or_condensing_design_and_control.is_sized_using_concident_load
elif fluid_loop.type == HEATING and fluid_loop_id in HW_fluid_loop_list:
      is_sized_using_coincident_load = fluid_loop.heating_design_and_control.is_sized_using_concident_load
...

In rule assertion at component level:
is_sized_using_coincident_load == True -> PASS
else:
FAIL.
```

Reply:
Updated.